.travis.yml

```diff
@@ -2,11 +2,10 @@
 language: ruby
 
 rvm:
-- 2.0.0
-- 2.1.6
-- 2.2.2
-- 2.4.1
-- 2.5.0
+- 2.4
```

Comment:
Why 2.3 is also dropped? Can we add it back?

Reply:
I dropped support for 2.3 since it's been end of life since 2019-03-31. I will add back the version.

Reply:
Added back support for 2.3 in e82f9e8.